lib/synx/project.rb

```diff
@@ -49,6 +49,12 @@ def set_options(options)
       end
 
       self.group_exclusions |= options[:group_exclusions] if options[:group_exclusions]
+      
+      if options[:no_sort_by_name]
```

Comment:
:+1: 
Of course it can be like this. I'm iOS developer so not so much experience in Ruby :)